### PR TITLE
Cosmetic changes, changed copyright

### PR DESCRIPTION
**Pull request by niden**
Hello!

* Type: code quality
* Link to issue: #13543 

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [ ] I wrote some tests for this PR.

Cosmetic changes

Thanks

### CODE REVIEW

**Comment by niden** [comment]
cc @sergeyklay

**Review by sergeyklay — DISMISSED** [review]

File: phalcon/mvc/modelinterface.zep [hunk 1]
```diff
@@ -167,89 +182,74 @@ interface ModelInterface
...
 
 	/**
-	 * Deletes a model instance. Returning true on success or false otherwise.
+	 * Sets the record's snapshot data.
+	 * This method is used internally to set snapshot data when the model was set up to keep snapshot data
+	 *
+	 * @param array columnMap
 	 */
-	public function delete() -> boolean;
+	public function setSnapshotData(array! data, columnMap = null);
```
Comment (virgofx) [inline phalcon/mvc/modelinterface.zep]: Should we have return void here?

File: phalcon/mvc/modelinterface.zep [hunk 2]
```diff
@@ -167,89 +182,74 @@ interface ModelInterface
...
 	/**
-	 * Returns related records based on defined relations
+	 * Allows to calculate a sum on a column that match the specified conditions
 	 *
-	 * @param array arguments
+	 * @param array parameters
+	 * @return double
 	 */
-	public function getRelated(string alias, arguments = null) -> <ResultsetInterface>;
+	public static function sum(parameters = null);
```
Comment (virgofx) [inline phalcon/mvc/modelinterface.zep]: Return type?

File: phalcon/mvc/modelinterface.zep [hunk 3]
```diff
@@ -25,130 +25,145 @@ use Phalcon\Mvc\Model\TransactionInterface;
 interface ModelInterface
 {
 	/**
-	 * Returns the models meta-data service related to the entity instance.
+	 * Appends a customized message on the validation process
 	 */
-	public function getModelsMetaData() -> <MetaDataInterface>;
+	public function appendMessage(<MessageInterface> message);
```
Comment (virgofx) [inline phalcon/mvc/modelinterface.zep]: return type: void?

File: phalcon/mvc/modelinterface.zep (hunk 2 above)
Comment (niden) [reply phalcon/mvc/modelinterface.zep]: @virgofx That will come on the next pass. Still checking them. I found for instance that the `getEventsManager` was protected....

File: unit-tests/ModelsMetadataTest.php [hunk 4]
```diff
@@ -180,7 +180,7 @@ protected function _executeTests($di)
 			'ciudad_id' => 0,
 			'creado_at' => 1,
 			'cupo' => 3,
-			'estado' => 18,
+			'estado' => 5,
```
Comment (sergeyklay) [inline unit-tests/ModelsMetadataTest.php]: @niden Why this test was changed? Did you change the logic?

**Comment by niden** [comment]
@sergeyklay It is ready now for review

**Review by sergeyklay — APPROVED** [review]

**Pull request merged**